pkg/cli/repositories.go

```diff
@@ -0,0 +1,447 @@
+package cli
+
+import (
+	"bytes"
+	"crypto/md5"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/vorteil/vorteil/pkg/vpkg"
+)
+
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2020 vorteil.io Pty Ltd
+ **/ //
+
+var repositoriesCmd = &cobra.Command{
+	Use:   "repositories",
+	Short: "Interact with vorteil repositories",
+}
+
+var keysCmd = &cobra.Command{
+	Use:   "keys",
+	Short: "Create, List and Delete keys for authentication with Vorteil Repositories",
+}
+
+var defaultKeyCmd = &cobra.Command{
+	Use:   "default",
+	Short: "Prints what the current default is",
```

Comment:
Change Short to something like `View or change the default repository key` and add logic so you can set default like `vorteil repositories keys default KEY_NAME`

Reply:
Still cannot set default key
```sh
$ ./borteil repositories keys default key1
write /home/jon/.vorteil/repository-keys/key1: copy_file_range: bad file descriptor
```

Reply:
Think i fixed this in the next upload